### ISSUE TITLE
Brush: refine painted edge so it tapers to straight edge at 2000px.

### ISSUE THREAD

**Issue by patrickmckowen** [JDRF/spirit#256]

**Comment by ismaeltoure7** [comment]
@patrickmckowen 
Here's a tapered approach to the ends of the brushes. Let me know if this works: https://share.goabstract.com/7f56da2f-bcb4-4988-a100-1800c0eca200

**Comment by patrickmckowen** [comment]
@ismaeltoure7 left a comment for refinement of this asset.  @joesnellpdx if you are still waiting on the new asset, I would just move forward with the existing one and we can update it with new asset when it's ready/we have time?

**Comment by joesnellpdx** [comment]
@patrickmckowen I used the new one from above per this in slack: https://jdrfmarcom.slack.com/archives/C6BA0C2F5/p1536615800000100?thread_ts=1536603051.000100&cid=C6BA0C2F5. 

You can see it in the new PR: https://github.com/JDRF/spirit/pull/262

Can switch back if needed - have both sets.
